File: easy/eit/eit.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

# started at: Sunday, 17. November 2013 02:30PM
import sys


iteration,divisor = map(int,sys.stdin.readline().split())
count = 0
inputs = sys.stdin.readlines()

for inp in inputs:
    if int(inp) % divisor == 0:
        count += 1
print count

File: easy/eit/eit.c
/*
    started at: Sunday, 17. November 2013 02:30PM
*/

#include <stdio.h>

int main(int argc, char const *argv[])
{
    int iterations;
    int divisor;
    int number;

    int count = 0;
    
    scanf("%d",&iterations);
    scanf("%d",&divisor);

    for (int i = 0; i < iterations; ++i)
    {   
        scanf("%d",&number);
        if (number % divisor==0)
        {
            count++;
        }
    }
    printf("%d\n",count);
    return 0;
}